pkgs/by-name/ar/ares/package.nix

```
@@ -2,58 +2,81 @@
   lib,
   SDL2,
   alsa-lib,
-  autoPatchelfHook,
+  apple-sdk_12,
+  cmake,
   fetchFromGitHub,
   gtk3,
   gtksourceview3,
+  gnumake,
   libGL,
   libGLU,
   libX11,
   libXv,
   libao,
-  libicns,
   libpulseaudio,
   librashader,
+  ninja,
+  moltenvk,
   openal,
   pkg-config,
   stdenv,
+  stdenvNoCC,
   udev,
   vulkan-loader,
-  which,
   wrapGAppsHook3,
+  zlib,
 }:
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "ares";
-  version = "141";
+  version = "142";
 
   src = fetchFromGitHub {
     owner = "ares-emulator";
     repo = "ares";
-    rev = "v${finalAttrs.version}";
-    hash = "sha256-iNcoNdGw/DfYc9tsOGsPYoZLhVwNzJe8bVotx6Rl0j4=";
+    rev = "v142";
```

I personally would keep using `finalAttrs.version`, but maybe I have missed a new convention.

```suggestion
    tag = "v${finalAttrs.version}";
```

---

That was changed because initially I had moved the source and slang-shaders out of the derivation, and when I copied it back instead of doing what I was doing I forgot to change it back. Will fix in a sec.